src/debugger/cli-debugger.c

```diff
@@ -1286,6 +1287,7 @@ void CLIDebuggerAttachBackend(struct CLIDebugger* debugger, struct CLIDebuggerBa
 }
 
 bool CLIDebuggerTabComplete(struct CLIDebugger* debugger, const char* token, bool initial, size_t tokenLen) {
+	UNUSED(initial);
```

Comment:
This warning and a few others were intentionally left in to remind me that there was additional work to be done in those functions. A simple UNUSED just abdicates that work instead of doing the work properly.

Reply:
reverted